backend/src/test/java/gov/cdc/usds/simplereport/service/UploadServiceTest.java

```diff
@@ -145,6 +145,20 @@ void testMalformedCSV() {
     assertThat(error.getMessage()).contains("Not enough column values: expected 21, found 1");
   }
 
+  @Test
+  void testInvalidRace() {
+    // GIVEN
+    InputStream inputStream = loadCsv("test-upload-no-race.csv");
+
+    // WHEN
+    var error =
+        assertThrows(
+            IllegalArgumentException.class, () -> this._service.processPersonCSV(inputStream));
+
+    // THEN
+    assertThat(error.getMessage()).isEqualTo("Error on row 1; Race is required.");
+  }
```

Comment:
This is great! thanks for adding this, can you also add a test for when there is a value but its not in our RACES map

Reply:
Good call, see 6bb1b16 👍